third_party/move/move-model/src/ast.rs

```diff
@@ -859,10 +885,11 @@ impl ExpData {
     {
         let mut found = false;
         self.visit(&mut |e| {
```

Comment:
It's debatable, since I suspect most action will be in the leaves of the tree, but there could be slightly less work with `visit_pre_order()` (was `visit_top_down`).